### ISSUE TITLE
using spawn eggs on other players plots

### ISSUE THREAD

**Issue by PhanaticD** [IntellectualCrafters/PlotSquared#366]
You can use a mob spawn egg on another players plot as long as you use it on a water block

**Comment by Sauilitired** [comment]
@boy0001 I don't quite understand ho this is happening ;o

```java
if ((type == Material.WATER || type == Material.STATIONARY_WATER) && event.getMaterial() == Material.MONSTER_EGG) {
  if (!plot.isAdded(player.getUniqueId())) {
    event.setCancelled(true);
  }
}
```

**Comment by c0wg0d** [comment]
It works anywhere, not just on water blocks.  Using PlotSquared 2.11.27 and Spigot 1.8.7 with WorldGuard 6.1.

**Comment by boy0001** [comment]
Fixed as of 3a1577469b1647ef56b1c3a3f7b65de5e7d4994b
